R/viz_carousel.R

```diff
@@ -8,11 +8,15 @@
 viz_carousel <- function(data, height = 18.3, width = 36.8) {
 
   if( is.character(data) ) { 
-    return( list(error = data) )
+    return( 
```

Comment:
Deze 2 condities komen steeds weer terug en betekenen dat helemaal niets berekend kan worden. We kunnen deze ook naar csp_dna.R verplaatsen. En ipv hier een svg te doen, wel gewoon text via de oude manier, zodat een array van errors wordt teruggegeven. 

Maar dit kan ook in een volgende MR.

Reply:
Oh wacht, beter niet wachten, gezien hij nu ubherhaupt de keys missen. En bij de carousel is het dan wel lastig wat we hier mee moeten.

Reply:
Ook falen nu de tests, aangezien die nog steeds de errors array verwachten. 

En de positieve specs kijken vooral of er een svg wordt teruggegeven, niet wat er in staat, dus die moeten ook wat specieker kijken.